operator/v1/types_openshiftcontrollermanager.go

```diff
@@ -28,8 +28,23 @@ type OpenShiftControllerManager struct {
 
 type OpenShiftControllerManagerSpec struct {
 	OperatorSpec `json:",inline"`
+
+	// imageRegistryAuthTokenType directs the openshift-controller-manager to use either a
+	// legacy,(unbound, long-lived) service acccount tokens or a bound service account
+	// token when generating image pull secrets for the integrated image registry.
+	// +kubebuilder:default=Legacy
```

Comment:
Have you tried spinning up a cluster, upgrading it to this CRD, reading it, then removing the default from the CRD spec, and confirming that the resource has got the default as applied? If this is the mechanism for making sure that old clusters upgrade and retain the value, I'm not sure this is going to be effective, you may need a controller to set the value in 4.15

Reply:
The idea is to backport https://github.com/openshift/api/pull/1758 to 4.15. This PR would be the 4.16 follow-up.